applications/trunkstore/src/ts_util.erl

```diff
@@ -108,6 +108,7 @@ lookup_did(DID, AccountId) ->
             Resp;
         {'error', 'not_found'} ->
             Options = [{'key', DID}],
+            CacheProps = [{'origin', [{'db', wnm_util:number_to_db_name(DID), DID}, {'type', <<"number">>}]}],
```

Comment:
You need to update trunkstore_sup's origin bindings to include `{'type', <<"number">>}`, similar to stepswitch_sup.